src/test/jtx/TrustedPublisherServer.h

```diff
@@ -150,32 +447,52 @@ class TrustedPublisherServer
             return;
 
         static int id_ = 0;
-        std::thread{lambda{++id_, *this, std::move(sock_)}}.detach();
+        std::thread{lambda{++id_, *this, std::move(sock_), useSSL_}}.detach();
```

Comment:
Not sure we want to move `sock_` as `async_accept` needs it.

Reply:
I think what's happening is that sock_ is being stolen/moved for each session and then a new sock is effectively created at that moved-from location by the next accept. I don't necessarily like it, but I wasn't motivated to change it either.